src/hotspot/os_cpu/linux_aarch64/vm_version_linux_aarch64.cpp

```diff
@@ -185,7 +185,13 @@ static bool read_fully(const char *fname, char *buf, size_t buflen) {
   assert(buflen >= 1, "invalid argument");
   int fd = os::open(fname, O_RDONLY, 0);
   if (fd != -1) {
+    PRAGMA_DIAG_PUSH
+    PRAGMA_NONNULL_IGNORED
+    // Use pragma to suppress false positive gcc compile warning which maybe be gcc bug
+    // recorded by https://gcc.gnu.org/bugzilla/show_bug.cgi?id=87489. The false positive warning
+    // not seen with vanilla gcc release since maybe include some distro-specific gcc patch.
```

Comment:
The wording of the comment is a bit awkward and ungrammatical.  Suggest something like:

Suppress false positive gcc warning, which may be an example of
https://gcc.gnu.org/bugzilla/show_bug.cgi?id=87489
The warning also hasn't been seen with vanilla gcc release, so may also
involve some distro-specific gcc patch.

Reply:
Apologize for my poor English. Thanks your advice.